### ISSUE TITLE
(possible) Regression from 7.1.3 to 7.1.4 for influxdb adaptor

### ISSUE THREAD

**Issue by systemcrash** [grafana/grafana#27169]
**What happened**:

`if_octets` and `if_packets` graphs fail to draw (no data returns) on 714. Work fine on 713. 

Compare. 

713:
![713 Kapture 2020-08-24 at 17 33 27](https://user-images.githubusercontent.com/647633/91065269-8dcc0c00-e630-11ea-81d6-3e40ec958428.gif)

query inspector 713 response:
```
response:Object
results:Array[2]
0:Object
statement_id:0
series:Array[1]
0:Object
name:"interface_rx"
columns:Array[2]
0:"time"
1:"non_negative_derivative"
values:Array[1439]
0:Array[1595691000000,19859.95071759277]
1:Array[1595692800000,33004.9860401233]
.........etc......
```

714:
![714 Kapture 2020-08-24 at 17 35 43](https://user-images.githubusercontent.com/647633/91065308-99b7ce00-e630-11ea-99d5-090f3dab842b.gif)

query inspector 714 response:
```
response:Object
results:Array[2]
0:Object
statement_id:0
1:Object
statement_id:1
```

Panel expressions:
Octets: A
`SELECT non_negative_derivative(mean("value"), 1s) FROM "autogen"."interface_rx" WHERE ("host" =~ /^$devname$/ AND "type" = 'if_octets' AND "instance" =~ /^$interface$/) AND $timeFilter GROUP BY time($__interval) fill(none)`

Octets: B
`SELECT non_negative_derivative(mean("value"), 1s) FROM "autogen"."interface_tx" WHERE ("host" =~ /^$devname$/ AND "type" = 'if_octets' AND "instance" =~ /^$interface$/) AND $timeFilter GROUP BY time($__interval) fill(none)`

Packets: A
`SELECT non_negative_derivative(mean("value"), 1s) FROM "autogen"."interface_rx" WHERE ("host" =~ /^$devname$/ AND "type" = 'if_packets' AND "instance" =~ /^$interface$/) AND $timeFilter GROUP BY time($__interval) fill(none)`

Packets: B
`SELECT non_negative_derivative(mean("value"), 1s) FROM "autogen"."interface_tx" WHERE ("host" =~ /^$devname$/ AND "type" = 'if_packets' AND "instance" =~ /^$interface$/) AND $timeFilter GROUP BY time($__interval) fill(none)`

etc

I suspect [this PR](https://github.com/grafana/grafana/pull/27012) which went into 7.1.4

**What you expected to happen**:

worky

**How to reproduce it (as minimally and precisely as possible)**:

Upgrade working installation (713) to 714:

(edit docker-compose.yaml)
`image: grafana/grafana:7.1.3` --> `image: grafana/grafana:7.1.4`
`docker-compose kill grafana ; docker-compose up -d`
Reload page. 

Bleh.

**Anything else we need to know?**:

**Environment**:
- Grafana version: v7.1.3 (5723d951af) / v7.1.4 (82a235b54c)
- Data source type & version: influxdb (built in) - InfluxDB docker image self reports: 

X-Influxdb-Version | 1.7.9
-- | --

- OS Grafana is installed on: docker
- User OS & Browser: Catalina 10.15.6 (19G2021) / Firefox 79.0 (64-bit)
- Grafana plugins: OpenNMS Flow / `GF_INSTALL_PLUGINS=opennms-helm-app`
- Others:

**Comment by torkelo** [comment]
duplicate of #27119

fixed in 7.1.5 (release today)

**Comment by systemcrash** [comment]
Problem confirmed as resolved in 7.1.5
